akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/MaxThroughputSpec.scala

```diff
@@ -55,6 +54,12 @@ object MaxThroughputSpec extends MultiNodeConfig {
            # See akka-remote-tests/src/test/resources/aeron.properties
            #advanced.embedded-media-driver = off
            #advanced.aeron-dir = "target/aeron"
+           
+           #advanced.compression {
+           #  enabled = on
+           #  actor-refs.enabled = on
+           #  manifests.enabled = on
+           #}
          }
        }
```

Comment:
and it was a major drop in the first table PR, we should get back to where we were. Potentially it should be better (we send less data) than before. See build 145:
https://jenkins.akka.io:8498/job/akka-artery-perf/plot/MaxThroughputSpec/
https://jenkins.akka.io:8498/job/akka-artery-perf/plot/LatencySpec/